client/java/src/main/java/com/linecorp/centraldogma/client/Latest.java

```diff
@@ -63,12 +69,12 @@ public boolean equals(Object o) {
             return false;
         }
         final Latest<?> that = (Latest<?>) o;
-        return Objects.equals(revision, that.revision) && Objects.equals(value, that.value);
+        return revision.equals(that.revision) && Objects.equals(value, that.value);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(revision) * 31 + Objects.hashCode(value);
+        return revision.hashCode() * 31 + Objects.hashCode(value);
     }
 
     @Override
```

Comment:
nit: `omitNullValues()`?

Reply:
I think we should not omit null in this case because we should tall the caller that the value is null.